samza-operator/src/test/java/org/apache/samza/operators/impl/TestSimpleOperatorImpl.java

```diff
@@ -20,26 +20,31 @@
 
 import org.apache.samza.operators.TestMessage;
 import org.apache.samza.operators.TestOutputMessage;
-import org.apache.samza.operators.internal.Operators.StreamOperator;
+import org.apache.samza.operators.functions.FlatMapFunction;
+import org.apache.samza.operators.spec.StreamOperatorSpec;
 import org.apache.samza.task.MessageCollector;
 import org.apache.samza.task.TaskCoordinator;
 import org.junit.Test;
 
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.function.Function;
 
-import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 
 public class TestSimpleOperatorImpl {
```

Comment:
I noticed that you have changed the SimpleOperatorImpl to StreamOperatorImpl. It would be nice to keep the test class name consistent as well.